router/src/server.rs

```diff
@@ -222,8 +222,8 @@ impl<'a, B: BatchType> BatchConfigValidator<'a, B> {
             self.batch_type.prefill_weight(&single_request_stats, 1);
         if max_batch_weight < single_request_prefill_weight {
             panic!(
-                "max_batch_weight ({}) not large enough for (prefill) max_sequence_length ({})",
-                max_batch_weight, max_sequence_length
+                "The provided max_sequence length ({}) results in a prefill batch weight that exceeds the estimated capacity (max_batch_weight: {})",
```

Comment:
Nice!
Could this be even simpler in both cases like: `The provided max_sequence length ({}) exceeds available GPU memory`?

Reply:
I'm just thinking that including `max_batch_weight` at all will still confuse users since it's not something they control

Reply:
Yes, I agree with you from a UX perspective. I also don't want to handle confused issue reports :P. But at the same time I would like to be able to see the value it's comparing against for debugging purposes. What do you think?

Reply:
That's why I changed the wording to "estimated capacity" but maybe I can omit the reference to `max_batch_weight` entirely

Reply:
Yeah, that's a good middle ground. RIght now I think it looks like you removed the reference to `max_batch_weight` in the first print but not the second

Reply:
Oops